interval/lift/testdata/in.axt

```diff
@@ -0,0 +1,8 @@
+0 chr1 101 120 chr2 201 220 + 1000
+ACTGACTGACTGATGCACTG
+ACTGACTCACTGATGGACTT
+
+0 chr1 101 120 chr2 201 220 - 1000
```

Comment:
Note to self: In a real axt, this would not have been the same region in 2 blocks. Imagine these are separate axt files or diff chr, e.g. block 1 is chr1 101 120, block 2 is chr3 101 120